docs/index.rst

```diff
@@ -1,19 +1,41 @@
+**************
```

Comment:
since this is essentially the README, you should just `.. include::` it (you can even set marks where you want the include to start and end)